maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -235,6 +235,11 @@ class AndroidDriver(
         }
     }
 
+    override fun shake() {
+        dadb.shell("emu sensor set acceleration 100:100:100")
+        dadb.shell("emu sensor set acceleration 0:0:0")
```

Comment:
have you tested this? 

I've seen something like this on a SO thread:
```
adb emu sensor set acceleration 100:100:100; sleep 1; adb emu sensor set acceleration 0:0:0
```

curious how we make sure this sort of function actually works